assets/stylesheets/_summary.scss

```diff
@@ -46,7 +46,7 @@
 $hmpo-summary-list: true !default;
 @if $hmpo-summary-list {
     .govuk-summary-list {
-        @extend .govuk-summary-list.govuk-summary-list--confirmation;
+        @extend .govuk-summary-list, .govuk-summary-list--confirmation;
```

Comment:
in this case we could probably just do 
```
@extend .govuk-summary-list--confirmation;
```

Which appears to work the same and compiles with `sass` as well as `node-sass`

Reply:
Awesome, I'll update the PR